### DISCUSSION TITLE
[Feature Request] Additional Row Panel Title Settings

### DISCUSSION THREAD

**Discussion by MarcDouguedroit** [grafana/grafana#50172] [Dashboards, Panels & Visualizations]
Rows can be used to create sections with similar data in dashboards grouping multiple data types. However, their titles are not put forward at the moment.

To improve dashboards and rows readability some additional options to the **row title** would be useful:
- **Text alignment:** Auto | Center
- **Color Mode:** None [ Text | Background
- **Text Size:** Value
- **Color**

This would allow us to display the row title in a different color in the middle of a screen, effectively splitting the dashboard into obvious sections.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
